src/main/java/org/broadinstitute/consent/http/ConsentApplication.java

```diff
@@ -368,8 +368,17 @@ private void initializeLiquibase(ConsentConfiguration config) throws LiquibaseEx
                 config.getDataSourceFactory().getPassword()
         );
         Database database = DatabaseFactory.getInstance().findCorrectDatabaseImplementation(new JdbcConnection(connection));
-        Liquibase liquibase = new Liquibase("changelog-master.xml", new ClassLoaderResourceAccessor(), database);
+        Liquibase liquibase = new Liquibase(liquibaseFile(), new ClassLoaderResourceAccessor(), database);
         liquibase.update(new Contexts(), new LabelExpression());
     }
 
+    private String liquibaseFile() {
+        String changeLogFile = System.getenv("CONSENT_CHANGELOG_FILE");
+        if (changeLogFile == null || changeLogFile.trim().isEmpty()) {
+            changeLogFile = "changelog-master.xml";
+        }
+        LOGGER.info(changeLogFile);
```

Comment:
Can we expand this a little? Something like
```suggestion
        LOGGER.info("Initializing db with: " + changeLogFile);
```